src/mysql/populate.sql

```diff
@@ -101,3 +106,4 @@ INSERT INTO Comment (answer_id, body, author_id, date_time)
 VALUES
   (3, 'And how much does it cost?', 7, '2020-07-29 19:00:00.000000'),
   (3, '50$', 1, '2020-07-29 20:00:00.000000');
+  
```

Comment:
nit: Add newline at end of file.

Reply:
it does have a newline

Reply:
Oh, I think GitHub was marking it. Just dismiss this then :)